code/game/objects/items/storage/storage.dm

```diff
@@ -332,36 +392,38 @@
 			row_num = round((adjusted_contents-1) / 7) // 7 is the maximum allowed width.
 		slot_orient_objs(row_num, col_count, numbered_contents)
 
-//This proc return 1 if the item can be picked up and 0 if it can't.
-//Set the warning to stop it from printing messages
-/obj/item/storage/proc/can_be_inserted(obj/item/W as obj, warning = TRUE)
-	if(!istype(W) || (W.flags_item & NODROP))
+/**
+ * This proc returns TRUE if the item can be picked up and FALSE if it can't.
+ * Set the warning to stop it from printing messages
+**/
+/obj/item/storage/proc/can_be_inserted(obj/item/item, warning = TRUE)
```

Comment:
if you renamed other can be inserted  args as well to item ignore my comments about them, also remember to check all calls on this to see if they dont have a karg assignment to old W